src/containers/preview-modal.jsx

```diff
@@ -36,10 +36,7 @@ class PreviewModal extends React.Component {
         this.props.onViewProject();
     }
     supportedBrowser () {
-        if (platform.name === 'IE') {
-            return false;
-        }
-        return true;
+        return !['IE', 'Opera', 'Opera Mini', 'Silk', 'Vivaldi'].includes(platform.name);
```

Comment:
Note - Currently IE doesn't actually get here - it fails earlier and gets handled in the error-boundary.